Add CORS requirement for custom fonts (#5685)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/drag_and_drop/dnd_email_style_settings.md

With custom fonts, you can manually add a web font for branding consistency across various email platforms. You can add one custom font per section. 

Before adding a custom font, check that the font meet the following requirements:

> - CORS must be enabled on the server that provides the custom font. This is usually handled by your IT team.

Reviewer: Should we link to Mozilla docs? https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
```suggestion
- CORS must be enabled on the server that provides the custom font file. This is usually handled by your IT team.
```

> - CORS must be enabled on the server that provides the custom font file. This is usually handled by your IT team.

Reviewer: ```suggestion
- CORS must be enabled on the server that provides the custom font file. This is usually handled by your IT team.
  - The file must have the header: `Access-Control-Allow-Origin: *`
```

Something like this, though this is pretty technical, so relying on your double check!

- The file URL must point to a .css file (not a .woff, .otf, etc.).
- The custom font name must match the name of the font face in the .css file.

To add a custom font:

1. Click **Add a custom font** below the **Default Font Name** of the styling section.

> 2. For the **Font Name** field, enter the same font name as your custom font source file. Ensure that the name is capitalized and spaced correctly. 

Reviewer: ```suggestion
2. For the **Font Name** field, enter the same font name that appears in your custom font source file. Ensure that the name is capitalized and spaced correctly. 
```

With custom fonts, you can manually add a web font for branding consistency across various email platforms. You can add one custom font per section. 

Before adding a custom font, check that the custom font file meets the following requirements:

- CORS must be enabled on the server that provides the custom font file. This is usually managed by your IT team. 

>   - The custom font file must have the header: `Access-Control-Allow-Origin: *`

Reviewer: should this have a period?

> - CORS must be enabled on the server that provides the custom font file. This is usually managed by your IT team. 
>   - The custom font file must have the header: `Access-Control-Allow-Origin: *`
> - The file URL must point to a CSS file (not WOFF, OTF, etc.).
> - The custom font name must match the name of the font face in the CSS file.

Author: ```suggestion
- CORS must be enabled on the server that provides the custom font file. This is usually managed by your IT team. 
  - The custom font file must have the header: `Access-Control-Allow-Origin: *`
- The file URL must point to a CSS file (not WOFF, OTF, etc.)
- The custom font name must match the name of the font face in the CSS file
```
